reliability-engineering/pipelines/build-govsvc-docker.yml

```diff
@@ -0,0 +1,52 @@
+resources:
+- name: govsvc-aws-ruby
+  type: docker-image
+  source:
+    username: ((dockerhub-username))
+    password: ((dockerhub-password))
+    repository: govsvc/aws-ruby
```

Comment:
What will happen to https://hub.docker.com/r/gdsre/aws-ruby?

Reply:
we'll migrate away from it, change usages to this one instead